### PR TITLE
Control classes

### PR DESCRIPTION
**Pull request by jaclynperrone**
This commit does a few things:

- Minor updates to README
- Adds classes to control buttons to make them more customizable across other projects
- Updates test to look for the zoom in/out button `className` instead of `textContent`

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/pileup.js/199)

### CODE REVIEW

File: src/Controls.js [hunk 1]
```diff
@@ -74,10 +74,11 @@ var Controls = React.createClass({
         </select>{' '}
         <input ref='start' type='text' />–
         <input ref='stop' type='text' />{' '}
-        <button>Go</button>{' '}
-
-        <button onClick={this.zoomOut}>-</button>{' '}
-        <button onClick={this.zoomIn}>+</button>
+        <button className='btn-submit'>Go</button>{' '}
```
Comment (danvk) [inline src/Controls.js]: Is the `btn` prefix a bootstrap thing? You can also style it using `button.submit` if it's important to make it clear that this is a button.

**Comment by danvk** [comment]
I'm curious about the class names, but other than that looks good to merge. Thanks!

File: src/Controls.js (hunk 1 above)
Comment (jaclynperrone) [inline src/Controls.js]: Nope, not a bootstrap thing. Was just something I made up/something I usually use on buttons.

**Comment by danvk** [comment]
Feel free to merge!

**Pull request merged**